### ISSUE TITLE
Tuya zigbee temperature & Humity sensor IH-K009

### ISSUE THREAD

**Issue by yvoren** [JohanBendz/com.tuya.zigbee#237]
Dear Johan,

I bought a Tuya zigbee temperature & Humity sensor, however i cant ad it to my homey device with your app.
The product model is IH-K009. Is it possible to add this product to your homey app ?

Tanks in advanced

Greetings, René 
![A8749E3C-1406-47A9-A351-A3F7C7CD2D01](https://user-images.githubusercontent.com/60089671/198229329-88b68970-5101-40f9-8dfc-a7c1a2e57f04.jpeg)

**Comment by bartek0000** [comment]
I also bought the same one and had issues adding it to Homey. I installed the Experimental version of the Tuya App and was able to add the device. But now the device is Unavailable. I have added two more, but i get the same response from all of them. 
![Screenshot_20221027-211937_Homey](https://user-images.githubusercontent.com/50036220/198379763-8c9c5279-db6e-4c53-81b8-70ca4a865316.jpg)
![Screenshot_20221027-212418_Homey](https://user-images.githubusercontent.com/50036220/198380409-5051a251-d150-4790-9d6e-4c7b2b3172e5.jpg)

And also thanks in advance🙂
